app/views/catalog/_show_documents.html.erb

```diff
@@ -13,42 +13,11 @@
           <table id="files-table" class="table">
             <thead>
               <tr>
-                <th scope="col" nowrap="nowrap"><span>#</span></th>
-                <th scope="col" nowrap="nowrap"><span>Filename</span></th>
-                <% if @document.data_source == "dataspace" %>
-                  <th scope="col"><span>Description</span></th>
-                <% end %>
-                <th scope="col" nowrap="nowrap"><span>Filesize</span></th>
+                  <th>Filename</th>
+                  <th>Size</th>
               </tr>
```

Comment:
We ditched the columns with the sequence number and the description for each file since we don't show those in PDC Describe. (Plus we don't have file description in PDC Describe, this was for DataSpace records but we are moving away from that)